### PR TITLE
Yet more spkg_configure for standard python packages

### LINKED ISSUE #36301: build/pkgs/toml doesn't have install info - remove?
### Steps To Reproduce

that's all it has:

```
$ ls -R build/pkgs/toml/
build/pkgs/toml/:
distros

build/pkgs/toml/distros:
arch.txt  debian.txt  fedora.txt  gentoo.txt  opensuse.txt
```

### Expected Behavior

it should not be there

### Actual Behavior

it is there. Remove?

### Additional Information

_No response_

### Environment

```markdown
all
```

### Checklist

- [X] I have searched the existing issues for a bug report that matches the one I want to file, without success.
- [X] I have read the documentation and troubleshoot guide

### PR DESCRIPTION
**Pull request by dimpase**
This is a continuation of #36256 

- Fixes #36301
- Fixes https://groups.google.com/g/sage-release/c/1wOBmhvNJqc/m/Jk14VAbjBAAJ (hence marked critical)

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [x] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### :hourglass: Dependencies

- #36296: to use an up to date Sphinx
- #36267: updated ipympl, etc

Also, a Jupyter/Python issue was uncovered there, which might need work.

### CODE REVIEW

**Comment by dimpase** [comment]
`PYDEVD_DISABLE_FILE_VALIDATION=1` is needed for `./sage -t src/sage/repl/ipython_kernel/kernel.py` to pass, for some reason.

**Comment by dimpase** [comment]
Tested on Debian oldstable with pyenv-installed Python 3.11.5. Configured as
```
./configure --enable-system-site-packages --with-python=$(pyenv root)/shims/python3
```
I had to deviate from version constraint on `sphinx` by pip-installing version `7.0.0`,
and pip-installing setuptools version `63.4.3`.

Output of `pip freeze` is attached.
[pip_freeze_py3.11.5_with_pyenv.txt](https://github.com/sagemath/sage/files/12641070/pip_freeze_py3.11.5_with_pyenv.txt)

**Comment by dimpase** [comment]
@mkoeppe - I already mentioned this issue in #36256 - `nbconvert` could not find
templates. Judging from several `/doesnotexist` in the log here, are we poisoning the correct places? In this case it  is `$(pyenv root)/versions/3.11.5/share/jupyter/nbconvert/`, but it does not come up on the list of what `nbconvert` calls `root_dirs`.

```
[sagemath_doc_html-none] 0.00s - Debugger warning: It seems that frozen modules are being used, which may
[sagemath_doc_html-none] 0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
[sagemath_doc_html-none] 0.00s - to python to disable frozen modules.
[sagemath_doc_html-none] 0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
[sagemath_doc_html-none] [plot3d   ] Exception occurred:
[sagemath_doc_html-none] [plot3d   ]   File "/home/scratch/scratch2/dimpase/sage/pyenv/versions/3.11.5/lib/python3.11/site-packages/nbconvert/exporters/templateexporter.py", line 653, in get_template_names
[sagemath_doc_html-none] [plot3d   ]     raise ValueError(msg)
[sagemath_doc_html-none] [plot3d   ] ValueError: No template sub-directory with name 'script' found in the following paths:
[sagemath_doc_html-none] [plot3d   ]    /doesnotexist
[sagemath_doc_html-none] [plot3d   ]    /home/scratch/scratch2/dimpase/sage/sage/local/var/lib/sage/venv-python3.11/share/jupyter
[sagemath_doc_html-none] [plot3d   ]    /doesnotexist
[sagemath_doc_html-none] [plot3d   ]    /home/scratch/dimpase/.sage///local/share/jupyter
[sagemath_doc_html-none] [plot3d   ]    /usr/local/share/jupyter
[sagemath_doc_html-none] [plot3d   ]    /usr/share/jupyter
[sagemath_doc_html-none] [plot3d   ] The full traceback has been saved in /tmp/sphinx-err-15y95ur8.log, if you want to report the issue to the developers.
[sagemath_doc_html-none] [plot3d   ] Please also report this if it was a user error, so that a better error message can be provided next time.
[sagemath_doc_html-none] [plot3d   ] A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
[sagemath_doc_html-none] Error building the documentation.
[sagemath_doc_html-none] Traceback (most recent call last):
[sagemath_doc_html-none]   File "<frozen runpy>", line 198, in _run_module_as_main
[sagemath_doc_html-none]   File "<frozen runpy>", line 88, in _run_code
[sagemath_doc_html-none]   File "/home/scratch/scratch2/dimpase/sage/sage/pkgs/sage-docbuild/sage_docbuild/__main__.py", line 503, in <module>
[sagemath_doc_html-none]     sys.exit(main())
[sagemath_doc_html-none]              ^^^^^^
[sagemath_doc_html-none]   File "/home/scratch/scratch2/dimpase/sage/sage/pkgs/sage-docbuild/sage_docbuild/__main__.py", line 499, in main
[sagemath_doc_html-none]     builder()
[sagemath_doc_html-none]   File "/home/scratch/scratch2/dimpase/sage/sage/pkgs/sage-docbuild/sage_docbuild/builders.py", line 818, in _wrapper
[sagemath_doc_html-none]     getattr(DocBuilder, build_type)(self, *args, **kwds)
[sagemath_doc_html-none]   File "/home/scratch/scratch2/dimpase/sage/sage/pkgs/sage-docbuild/sage_docbuild/builders.py", line 162, in f
[sagemath_doc_html-none]     runsphinx()
[sagemath_doc_html-none]   File "/home/scratch/scratch2/dimpase/sage/sage/pkgs/sage-docbuild/sage_docbuild/sphinxbuild.py", line 327, in runsphinx
[sagemath_doc_html-none]     sys.stderr.raise_errors()
[sagemath_doc_html-none]   File "/home/scratch/scratch2/dimpase/sage/sage/pkgs/sage-docbuild/sage_docbuild/sphinxbuild.py", line 263, in raise_errors
[sagemath_doc_html-none]     raise OSError(self._error)
[sagemath_doc_html-none] OSError: Exception occurred:
[sagemath_doc_html-none]
```

**Comment by mkoeppe** [comment]
this would need repro - #32531

**Comment by mkoeppe** [comment]
Using system jupyter = #30306

**Comment by dimpase** [comment]
I tried running make with 
```diff
--- a/src/bin/sage
+++ b/src/bin/sage
@@ -1041,10 +1041,10 @@ if [ "$1" = "-docbuild" -o "$1" = "--docbuild" ]; then
 
     # Issue #33650: Make sure that user configuration of Jupyter does not
     # shadow our sagemath kernel when jupyter-sphinx is invoked
-    export JUPYTER_CONFIG_DIR=/doesnotexist
-    export JUPYTER_CONFIG_PATH=/doesnotexist
-    export JUPYTER_DATA_DIR=/doesnotexist
-    export JUPYTER_PATH=/doesnotexist
+    #export JUPYTER_CONFIG_DIR=/doesnotexist
+    #export JUPYTER_CONFIG_PATH=/doesnotexist
+    #export JUPYTER_DATA_DIR=/doesnotexist
+    #export JUPYTER_PATH=/doesnotexist
 
     # Redirect stdin from /dev/null. This helps with running TeX which
     # tends to ask interactive questions if something goes wrong. These
```
but to no avail. The same error (these poisoned env vars don't point to the right place, anyway)

**Comment by dimpase** [comment]
> this would need repro - #32531

not sure what you mean here; this might be related to `pyenv`, but people run into these missing templates stuff in other contexts too, see e.g. https://stackoverflow.com/questions/63958574/no-template-sub-directory-with-name-lab-found-in-the-following-paths

So it's more the general `nbconvert`'s API flakiness.

**Comment by mkoeppe** [comment]
We have instructions how to use system jupyter. https://doc.sagemath.org/html/en/installation/launching.html#setting-up-sagemath-as-a-jupyter-kernel-in-an-existing-jupyter-notebook-or-jupyterlab-installation
Does it work if you follow these instructions?

**Comment by dimpase** [comment]
> We have instructions how to use system jupyter. https://doc.sagemath.org/html/en/installation/launching.html#setting-up-sagemath-as-a-jupyter-kernel-in-an-existing-jupyter-notebook-or-jupyterlab-installation Does it work if you follow these instructions?

I followed these instructions and installed sage's jupyter kernel just fine, but it does not remedy the issue. It seems that it's largely orthogonal to actually running notebooks (`jupyter notebook`) - here we rather want to run nbconvert (`jupyter nbconvert`).

**Comment by mkoeppe** [comment]
But does nbconvert (= Jupyter menu items for exporting) work normally in the jupyter notebook set up in this way?

**Comment by dimpase** [comment]
and I tried running `jupyter nbconvert --to html` on a random Sage jupyter notebook, it worked fine.

**Comment by mkoeppe** [comment]
:)

**Comment by dimpase** [comment]
it might be `jupyter_sphinx` package is to blame here. We get the error running `sphinx`, and `jupyter_sphinx` invokes `nbconvert`, according to https://jupyter-sphinx.readthedocs.io/en/latest/

**Comment by dimpase** [comment]
here is the detailed error log

[sphinx-err-09_j4sn5.log](https://github.com/sagemath/sage/files/12641726/sphinx-err-09_j4sn5.log)

**Comment by orlitzky** [comment]
> `PYDEVD_DISABLE_FILE_VALIDATION=1` is needed for `./sage -t src/sage/repl/ipython_kernel/kernel.py` to pass, for some reason.

This was just merged into develop today

**Comment by mkoeppe** [comment]
> it might be `jupyter_sphinx` package is to blame here. We get the error running `sphinx`, and `jupyter_sphinx` invokes `nbconvert`, according to https://jupyter-sphinx.readthedocs.io/en/latest/

Does it invoke nbconvert as an executable, or does it invoke it as a Python package?

**Comment by dimpase** [comment]
nbconvert is not an executable by itself, it can be invoked either as `jupyter nbconvert`, or via a Python API.

One way or another, it appears to chase the correct directory at a number of places, among them `JUPYTER_PATH` and other similarly named Python globals (which Sage poisons, but that's another story - the correct path is not there anyway). This looks like a horrible hack. 
Anyhow we can probably work around this, by specifying the correct path in some (Jupyter ?) config, but where, I don't know.

**Comment by dimpase** [comment]
A quick reproducer:
```
$ ./sage --docbuild file=src/doc/en/reference/plot3d/sage/plot/plot3d/plot3d.rst html
[html     ] /home/dimpase/.sage/docbuild/plot3d/source/conf.py:6: DeprecationWarning: This module is deprecated. Use sage_docbuild.conf instead.
[html     ] See https://github.com/sagemath/sage/issues/33763 for details.
[html     ]   from sage.docs.conf import *
[html     ] writing output... 
[html     ] building [html]: targets for 2 source files that are out of date
[html     ] updating environment: [new config] 2 added, 0 changed, 0 removed
0.00s - Debugger warning: It seems that frozen modules are being used, which may
0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
[html     ] WARNING: autodoc: failed to import module 'plot3d'; the following exception was raised:
[html     ] No module named 'plot3d'
[html     ] Exception occurred:
[html     ]   File "/home/dimpase/.pyenv/versions/3.11.5/lib/python3.11/site-packages/nbconvert/exporters/templateexporter.py", line 653, in get_template_names
[html     ]     raise ValueError(msg)
[html     ] ValueError: No template sub-directory with name 'script' found in the following paths:
[html     ]     /doesnotexist
[html     ]     /home/dimpase/work/software/sage/local/var/lib/sage/venv-python3.11/share/jupyter
[html     ]     /doesnotexist
[html     ]     /home/dimpase/.sage//local/share/jupyter
[html     ]     /usr/local/share/jupyter
[html     ]     /usr/share/jupyter
[html     ] The full traceback has been saved in /tmp/sphinx-err-_f5y_fwb.log, if you want to report the issue to the developers.
[html     ] Please also report this if it was a user error, so that a better error message can be provided next time.
[html     ] A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Error building the documentation.
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/dimpase/work/software/sage/pkgs/sage-docbuild/sage_docbuild/__main__.py", line 503, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/dimpase/work/software/sage/pkgs/sage-docbuild/sage_docbuild/__main__.py", line 499, in main
    builder()
  File "/home/dimpase/work/software/sage/pkgs/sage-docbuild/sage_docbuild/builders.py", line 162, in f
    runsphinx()
  File "/home/dimpase/work/software/sage/pkgs/sage-docbuild/sage_docbuild/sphinxbuild.py", line 327, in runsphinx
    sys.stderr.raise_errors()
  File "/home/dimpase/work/software/sage/pkgs/sage-docbuild/sage_docbuild/sphinxbuild.py", line 263, in raise_errors
    raise OSError(self._error)
OSError: WARNING: autodoc: failed to import module 'plot3d'; the following exception was raised:
```

**Comment by dimpase** [comment]
@jasongrout - could you perhaps advise us on how to deal with this issue properly? It looks as if `nbconvert`, installed in pyenv, does not know where to look for templates. 

I thought that perhaps `export JUPYTER_PREFER_ENV_PATH=1` will do the trick, but no, it appears to have no effect. 
It's a mystery where `nbconvert` picks its list of `root_dirs` up, and how to configure it.

**Comment by mkoeppe** [comment]
Let's merge #36267 and lose the jupyter_packaging change

File: build/pkgs/meson_python/spkg-configure.m4 [hunk 1]
```diff
@@ -1,3 +1,3 @@
 SAGE_SPKG_CONFIGURE([meson_python], [
-  SAGE_PYTHON_PACKAGE_CHECK([meson-python])
+  SAGE_PYTHON_PACKAGE_CHECK([meson_python])
```
Comment (orlitzky) [inline build/pkgs/meson_python/spkg-configure.m4:2]: Did this cause a problem? I recall making the inverse change but not the reason why. The argument to `SAGE_PYTHON_PACKAGE_CHECK` should be the name of the python package (not the sage package), although both names seem to work when passed to `pkg_resources.require()`. In that regard `meson-python` more closely matches what's in e.g. `install-requires.txt`

Comment (mkoeppe) [reply build/pkgs/meson_python/spkg-configure.m4:2]: dashes don't work as part of environment variable names - see https://groups.google.com/g/sage-release/c/1wOBmhvNJqc/m/8IDtj6Y4BQAJ

Comment (mkoeppe) [reply build/pkgs/meson_python/spkg-configure.m4:2]: on the Python side, dashes and underscores in distribution names are interchangeable and get normalized as needed

Comment (orlitzky) [reply build/pkgs/meson_python/spkg-configure.m4:2]: Ok, it still works for me with the underscore FWIW.

**Comment by dimpase** [comment]
> @jasongrout - could you perhaps advise us on how to deal with this issue properly? It looks as if `nbconvert`, installed in pyenv, does not know where to look for templates.
> 
> I thought that perhaps `export JUPYTER_PREFER_ENV_PATH=1` will do the trick, but no, it appears to have no effect. It's a mystery where `nbconvert` picks its list of `root_dirs` up, and how to configure it.

There is actually a real bug, too - that directory chasing never finds the stuff needed. I think what's happening is that it's called only after an initial discovery attempt, looking at the current venv jupyter share dir (that is, `$SAGE_ROOT/local/var/lib/sage/venv-python3.11/share/jupyter`), fails.

I tried to unpoison `JUPYTER_PATH` and set it to `$(pyenv root)/versions/3.11.5/share/jupyter/nbconvert` - but it didn't do anything.
Whereas directly symlinking the latter in the current venv jupyter share dir does the trick.

**Comment by dimpase** [comment]
To use the current sphinx (i.e. version 7.1+), I need
```diff
--- a/src/sage/plot/contour_plot.py
+++ b/src/sage/plot/contour_plot.py
@@ -1391,11 +1391,9 @@ def implicit_plot(f, xrange, yrange, **options):
         raise ValueError("fill=%s is not supported" % options['fill'])
 
 
-@options(plot_points=100, incol='blue', outcol=None, bordercol=None,
-         borderstyle=None, borderwidth=None, frame=False, axes=True,
-         legend_label=None, aspect_ratio=1, alpha=1)
-def region_plot(f, xrange, yrange, plot_points, incol, outcol, bordercol,
-                borderstyle, borderwidth, alpha, **options):
+@options(frame=False, axes=True, legend_label=None, aspect_ratio=1)
+def region_plot(f, xrange, yrange, plot_points=100, incol='blue', outcol=None, bordercol=None,
+                borderstyle=None, borderwidth=None, alpha=1, **options):
     r"""
     ``region_plot`` takes a boolean function of two variables, `f(x, y)`
     and plots the region where f is True over the specified
@@ -1658,6 +1656,13 @@ def region_plot(f, xrange, yrange, plot_points, incol, outcol, bordercol,
     from sage.structure.element import Expression
     from warnings import warn
     import numpy
+    options['plot_points'] = plot_points
+    options['incol'] = incol
+    options['outcol'] = outcol
+    options['bordercol'] = bordercol
+    options['borderstyle'] = borderstyle
+    options['borderwidth'] = borderwidth
+    options['alpha'] = alpha
 
     if not isinstance(f, (list, tuple)):
         f = [f]
```

as discussed on [sage-devel](https://groups.google.com/g/sage-devel/c/urKo1e6vbKs/m/UaSWhrx7BQAJ)

Should this be merged here, or in a separate PR?

**Comment by dimpase** [comment]
Anyhow, that `options` change needs work, as I get plotting doctests failing like
```
sage -t --random-seed=386503542288236691772684673281025390 src/sage/plot/contour_plot.py
**********************************************************************
File "src/sage/plot/contour_plot.py", line 1203, in sage.plot.contour_plot.implicit_plot
Failed example:
    implicit_plot(x**2 + y**2 == 2, (x,-3,3), (y,-3,3), fill=True, fillcolor='red')
Expected:
    Graphics object consisting of 2 graphics primitives
Got:
    verbose 0 (163: primitive.py, options) WARNING: Ignoring option 'incol'=red
    verbose 0 (163: primitive.py, options) WARNING: Ignoring option 'outcol'=None
    verbose 0 (163: primitive.py, options) WARNING: Ignoring option 'bordercol'=blue
    verbose 0 (163: primitive.py, options) WARNING: Ignoring option 'borderstyle'=None
    verbose 0 (163: primitive.py, options) WARNING: Ignoring option 'borderwidth'=None
    verbose 0 (163: primitive.py, options) WARNING: Ignoring option 'alpha'=1
    verbose 0 (163: primitive.py, options) 
    The allowed options for ContourPlot defined by a 150 x 150 data grid are:
        cmap           the name of a predefined colormap,
                            a list of colors, or an instance of a
                            matplotlib Colormap. Type: import matplotlib.cm;
                            matplotlib.cm.datad.keys()
                            for available colormap names.
        colorbar       Include a colorbar indicating the levels                    
        colorbar_optionsa dictionary of options for colorbars                       
        contours       Either an integer specifying the number of
                            contour levels, or a sequence of numbers giving
                            the actual contours to use.
        fill           Fill contours or not                                        
        label_options  a dictionary of options for the labels                      
        labels         show line labels or not                                     
        legend_label   The label for this item in the legend.                      
        linestyles     the style of the lines to be plotted                        
        linewidths     the width of the lines to be plotted                        
        plot_points    How many points to use for plotting precision               
        zorder         The layer level in which to draw                            
    <BLANKLINE>
...
```

**Comment by dimpase** [comment]
here is the patch which allows Sphinx 7.2.6 and produces no doctest errors
```diff
--- a/src/sage/plot/contour_plot.py
+++ b/src/sage/plot/contour_plot.py
@@ -1391,11 +1391,9 @@ def implicit_plot(f, xrange, yrange, **options):
         raise ValueError("fill=%s is not supported" % options['fill'])
 
 
-@options(plot_points=100, incol='blue', outcol=None, bordercol=None,
-         borderstyle=None, borderwidth=None, frame=False, axes=True,
-         legend_label=None, aspect_ratio=1, alpha=1)
-def region_plot(f, xrange, yrange, plot_points, incol, outcol, bordercol,
-                borderstyle, borderwidth, alpha, **options):
+@options(frame=False, axes=True, legend_label=None, aspect_ratio=1)
+def region_plot(f, xrange, yrange, plot_points=100, incol='blue', outcol=None, bordercol=None,
+                borderstyle=None, borderwidth=None, alpha=1, **options):
     r"""
     ``region_plot`` takes a boolean function of two variables, `f(x, y)`
     and plots the region where f is True over the specified
```

**Comment by dimpase** [comment]
> To use the current sphinx (i.e. version 7.1+), I need
> 
> 
> --- a/src/sage/plot/contour_plot.py
> +++ b/src/sage/plot/contour_plot.py

[...]

as discussed on [sage-devel](https://groups.google.com/g/sage-devel/c/urKo1e6vbKs/m/UaSWhrx7BQAJ)

Should this be merged here, or in a separate PR?
I've opened # 36295 for this.

**Comment by mkoeppe** [comment]
This needs more distro package information so it can be properly tested in CI

**Comment by dimpase** [comment]
> This needs more distro package information so it can be properly tested in CI

Perhaps we should test on CI using `pyenv`. Is there an easy way to generate `requirements.txt` from Sage configuration of Python packages (not post-build, but pre-build)? I don't know a distro that has all the upstream Python packages we provide assessible via its packaging system. 

One can also generate is manually and put it in `.github/`, but this is hard to maintain.

**Comment by mkoeppe** [comment]
> Perhaps we should test on CI using `pyenv`. [...]  I don't know a distro that has all the upstream Python packages we provide assessible via its packaging system.

What would be the point of that? Isn't the point of `--enable-system-site-packages` to use distro-provided Python packages? 
With pyenv you would be using pip packages, no?

**Comment by dimpase** [comment]
> > Perhaps we should test on CI using `pyenv`. [...]  I don't know a distro that has all the upstream Python packages we provide assessible via its packaging system.
> 
> What would be the point of that? Isn't the point of `--enable-system-site-packages` to use distro-provided Python packages? With pyenv you would be using pip packages, no?


Yes, certainly. Do distros patch a lot of pip packages? I suppose their Python packaging is not really different to what Sage does, with 99% of packages just a fixed version exact copy of what's on PyPI. 

Do we have another way to run Sage with as many pip packages as possible?


 @antonio-rojas @kiwifb @orlitzky - care to comment?

**Comment by mkoeppe** [comment]
> Do we have another way to run Sage with as many pip packages as possible?

Just put the package info in distros/
repology knows all the names.

**Comment by dimpase** [comment]
> > Do we have another way to run Sage with as many pip packages as possible?
> 
> Just put the package info in distros/ repology knows all the names.

Yes, I certainly understand how to enable more system-wide packages. However, how is this the answer to the question you quote here?

**Comment by mkoeppe** [comment]
Not sure what you are looking for; we may have been miscommunicating.
`cat build/pkgs/*/install-requires.txt` gives you a list of all the Python packages that we know about, in the format of a requirements.txt.
This does not pin the versions to the specific package versions of our packages (package-version.txt), of course. Do you want that?

**Comment by dimpase** [comment]
> Not sure what you are looking for; we may have been miscommunicating. `cat build/pkgs/*/install-requires.txt` gives you a list of all the Python packages that we know about, in the format of a requirements.txt. This does not pin the versions to the specific package versions of our packages (package-version.txt), of course. Do you want that?

Ah, right (some of these have version constraints though). This should suffice for setting up a pyenv-based CI to use PyPI packages, indeed.

**Comment by dimpase** [comment]
> This needs more distro package information so it can be properly tested in CI

this is done now.

**Comment by mkoeppe** [comment]
And slightly higher level: `sage-get-system-packages install-requires $(sage -package list :standard:)` etc.

**Review by mkoeppe — APPROVED** [review]

**Comment by dimpase** [comment]
In particular, one needs fresh furo to go with fresh sphinx, otherwise docs don't build (build still finishes without a fatal error, but no html gets generated).

**Comment by dimpase** [comment]
Is there anything to do here, still? I'm taking the liberty to turn this to positive review.

**Comment by mkoeppe** [comment]
It's in good enough shape for this experimental feature, so I agree with the positive review.

On `gentoo-python-3.11-standard-sitepackages` https://github.com/sagemath/sage/actions/runs/6264830191/job/17012722151, I see
```
  [sagemath_doc_html-none]     File "/sage/pkgs/sage-docbuild/sage_docbuild/builders.py", line 980, in get_new_and_updated_modules
  [sagemath_doc_html-none]       env = self.get_sphinx_environment()
  [sagemath_doc_html-none]             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  [sagemath_doc_html-none]     File "/sage/pkgs/sage-docbuild/sage_docbuild/builders.py", line 876, in get_sphinx_environment
  [sagemath_doc_html-none]       env = pickle.load(f)
  [sagemath_doc_html-none]             ^^^^^^^^^^^^^^
  [sagemath_doc_html-none]   AttributeError: Can't get attribute '_stable_repr_object' on <module 'sphinx.builders.html' from '/usr/lib/python3.11/site-packages/sphinx/builders/html/__init__.py'>
  [sagemath_doc_html-none]   
  [sagemath_doc_html-none]       Note: incremental documentation builds sometimes cause spurious
```

The other tested platforms look OK (despite the scary red)

**Comment by orlitzky** [comment]
Is that `spkg-configure.m4` for jupyter_packaging still being created?

**Comment by vbraun** [comment]
merge conflict

**Comment by mkoeppe** [comment]
> Is that `spkg-configure.m4` for jupyter_packaging still being created?

removed in 180e814

**Pull request merged**